serverless/package.json

```diff
@@ -1,8 +1,10 @@
 {
+  "name": "stellar-tss",
```

Comment:
This seems related to https://stackoverflow.com/a/60112825/834309. I could not successfully run the GH action without this.